Modules/Options/QuestieOptionsDefaults.lua

```diff
@@ -68,25 +68,27 @@ function QuestieOptionsDefaults:Load()
             autoTrackQuests = true,
             trackerShowCompleteQuests = true,
             collapseCompletedQuests = false,
+            hideCompletedObjectives = false,
             trackerShowQuestLevel = true,
             showBlizzardQuestTimer = false,
             hookTracking = true,
             trackerHeaderEnabled = true,
-            trackerHeaderAutoMove = false,
+            autoMoveHeader = true,
```

Comment:
This should be added to a new entry in the `Migration.lua` so existing configurations pick up this new name.